public/diory-demo-content/diograph.json

```diff
@@ -3,7 +3,7 @@
   "diograph": {
```

Comment:
This seems to be old format of diograph... Shouldn't be `rootId` there...

Using demo-content-room repository also here would be preferable.